### PR TITLE
Linux action

### PR DESCRIPTION
**Pull request by bam241**
some test around auto plugin build for linux

### CODE REVIEW

File: script/CubitExport-release.cmake [hunk 1]
```diff
@@ -0,0 +1,190 @@
+#----------------------------------------------------------------
+# Generated CMake target import file for configuration "Release".
+#----------------------------------------------------------------
```
Comment (gonuke) [inline script/CubitExport-release.cmake]: I think I asked this somewhere else, and I don't remember if it was answered.... is this file supposed to be in the SDK?  Wasn't it in the SDK in the past?

Comment (bam241) [reply script/CubitExport-release.cmake]: in this case nop, but in some yes. (small fix need to be done for macOS especially...)

**Comment by bam241** [comment]
@Shimwell I can;'t get GitHub to pass my secret URL into the action, any thoughts about what I am doing wrong ?

**Comment by bam241** [comment]
closing reopening from @svalinn

**Comment by shimwell** [comment]
> @Shimwell I can;'t get GitHub to pass my secret URL into the action, any thoughts about what I am doing wrong ?

Perhaps a secret be added to the github repo then access it like this in the bash ```${{ secrets.MY_TOKEN }}```

**Pull request closed without merging**